Formula/yarn.rb

```diff
@@ -1,26 +1,19 @@
-require "language/node"
-
 class Yarn < Formula
   desc "JavaScript package manager"
   homepage "https://yarnpkg.com/"
   url "https://yarnpkg.com/downloads/0.21.3/yarn-v0.21.3.tar.gz"
```

Comment:
What does `mirror` do? The Yarn download URL just redirects to the GitHub one (we use the yarnpkg.com URL so we're not tightly-coupled with Github's release system)